### DISCUSSION TITLE
Objeck 2.0?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#379] [General]
### Discussed in https://github.com/objeck/objeck-lang/discussions/367

<div type='discussions-op-text'>

<sup>Originally posted by **iahung2** September 24, 2023</sup>
This is the chance for you to break compatibility with old code and introduce new breaking changes. Let's call the development kit you normally release the Objeck SDK. The Objeck language is still at version 1.0.</div>

**Comment by objeck** [comment]
Not sure what you asking here...

**Reply by ghost** [reply]
Let's call the current Objeck as Objeck 1.0. You could start developing Objeck 2.0, where you could break compatibility with old code and introduce new breaking changes. This is the idea.

**Comment by objeck** [comment]
Adding new general frameworks and classes does not require a new version of Objeck. I would like to consider this if significant functionality is added, like a C++ integration framework.

**Reply by ghost** [reply]
Objeck 2.0 is about to change syntax. From the experience you have with Objeck 1.0, do you think it's already perfect and there is nothing to change?

**Reply by objeck** [reply]
Nope, nothing is perfect. The naming convention I use is ``year + month +  (# of release that month)``. For example ``9.1.0``.
